### ISSUE TITLE
Range support: "accept-ranges: bytes" not exposed without "Range:" request header

### ISSUE THREAD

**Issue by netwarex** [gentics/mesh#643]
## Gentics Mesh Version, operating system, or hardware.
- v0.29.6

## Operating System
- any

## JVM
- any

## Reproducer
```bash
curl -sv -H "Range: bytes=0-" https://demo.getmesh.io/api/v1/demo/nodes/df8beb3922c94ea28beb3922c94ea2f6/binary/image > /dev/null
```
```
< HTTP/2 206 
< accept-ranges: bytes
< content-disposition: attachment; filename=ford-gt.jpg
< content-encoding: identity
< content-range: bytes 0-530195/530196
< content-type: image/jpeg
< etag: "4728baff"
< webroot-response-type: binary
< content-length: 530196
< date: Mon, 04 Feb 2019 12:48:24 GMT
```

```bash
curl -sv https://demo.getmesh.io/api/v1/demo/nodes/df8beb3922c94ea28beb3922c94ea2f6/binary/image > /dev/null
```
```
< HTTP/2 200 
< content-disposition: attachment; filename=ford-gt.jpg
< content-encoding: identity
< content-type: image/jpeg
< etag: "4728baff"
< webroot-response-type: binary
< content-length: 530196
< date: Mon, 04 Feb 2019 12:48:31 GMT
```

## Expected behaviour and actual behaviour
The second curl request also should return the **accept-ranges: bytes** header

**Comment by Jotschi** [comment]
I suspect that only one line needs to be added to include the missing header. That should be all.

**Comment by netwarex** [comment]
Can we add this in the next 2 weeks? :)

**Comment by Jotschi** [comment]
I made a fix for this issue. We can only add this in the binary fields endpoint. It is currently not possible to add this for webroot as well.

**Comment by Jotschi** [comment]
Fixed with 0.31.2
